iothub/device/tests/TransportSettingsTests.cs

```diff
@@ -1,271 +1,333 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information
 
+using System;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Azure.Devices.Client.ApiTest;
+using Microsoft.Azure.Devices.Client.Transport.Mqtt;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
 namespace Microsoft.Azure.Devices.Client.Test
 {
-    using System;
-    using Microsoft.Azure.Devices.Client;
-    using Microsoft.Azure.Devices.Client.ApiTest;
-    using Microsoft.Azure.Devices.Client.Transport.Mqtt;
-    using Microsoft.VisualStudio.TestTools.UnitTesting;
-
     [TestClass]
     [TestCategory("Unit")]
     public class TransportSettingsTests
     {
-        const string LocalCertFilename = "..\\..\\Microsoft.Azure.Devices.Client.Test\\LocalNoChain.pfx";
-        const string LocalCertPasswordFile = "..\\..\\Microsoft.Azure.Devices.Client.Test\\TestCertsPassword.txt";
+        private const string LocalCertFilename = "..\\..\\Microsoft.Azure.Devices.Client.Test\\LocalNoChain.pfx";
+        private const string LocalCertPasswordFile = "..\\..\\Microsoft.Azure.Devices.Client.Test\\TestCertsPassword.txt";
 
-        [ExpectedException(typeof(ArgumentOutOfRangeException))]
         [TestMethod]
-        public void TransportSettingsTestTransportTypeAmqp()
+        [ExpectedException(typeof(ArgumentOutOfRangeException))]
+        public void AmqpTransportSettings_InvalidTransportTypeAmqp()
         {
-            var transportSetting = new AmqpTransportSettings(TransportType.Amqp);
+            _ = new AmqpTransportSettings(TransportType.Amqp);
         }
 
-        [ExpectedException(typeof(ArgumentOutOfRangeException))]
         [TestMethod]
-        public void TransportSettingsTestTransportTypeAmqpHttp()
+        [ExpectedException(typeof(ArgumentOutOfRangeException))]
+        public void AmqpTransportSettings_InvalidTransportTypeAmqpHttp()
         {
-            var transportSetting = new AmqpTransportSettings(TransportType.Http1);
+            _ = new AmqpTransportSettings(TransportType.Http1);
         }
 
-        [ExpectedException(typeof(ArgumentOutOfRangeException))]
         [TestMethod]
-        public void TransportSettingsTestTransportTypeAmqpTcpPrefetch0()
+        [ExpectedException(typeof(ArgumentOutOfRangeException))]
+        public void AmqpTransportSettings_UnderPrefetchCountMin()
         {
-            var amqpConnectionPoolSettings = new AmqpConnectionPoolSettings();
-            var transportSetting = new AmqpTransportSettings(TransportType.Amqp_Tcp_Only, 0, amqpConnectionPoolSettings);
+            _ = new AmqpTransportSettings(TransportType.Amqp_Tcp_Only, 0, new AmqpConnectionPoolSettings());
         }
 
         [TestMethod]
-        public void TransportSettingsTestTransportTypeAmqpWebSocket()
+        public void AmqpTransportSettings_Defaults()
```

Comment:
Can there be a better naming for this test?

Reply:
How's AmqpTransportSettings_DefaultPropertyValues?